src/scicookie/profiles/osl.yaml

```diff
@@ -1,100 +1,100 @@
 author_full_name:
-  default: Yor Briar
-  visible: true
+    default: Yor Briar
+    visible: true
 
 author_email:
-  default: yor.briar@spyx.family
-  visible: true
+    default: yor.briar@spyx.family
+    visible: true
 
 project_name:
-  default: My Super New Project
-  visible: true
+    default: My Super New Project
+    visible: true
 
 project_short_description:
-  visible: true
+    visible: true
 
 project_slug:
-  visible: true
+    visible: true
 
 package_slug:
-  visible: true
+    visible: true
 
 project_version:
-  visible: true
+    visible: true
 
 project_url:
-  visible: true
+    visible: true
 
 project_license:
-  # first choice is the default for the UI
-  choices:
-    - BSD 3 Clause
-    - MIT
-    - Apache Software License 2.0
-    - Other
-  visible: true
+    # first choice is the default for the UI
+    choices:
+        - BSD 3 Clause
+        - MIT
+        - Apache Software License 2.0
+        - Other
+    visible: true
 
 project_layout:
-  default : src
-  visible: false
+    default: src
+    visible: false
 
 build_system:
-  visible: true
+    visible: true
 
 command_line_interface:
-  visible: true
+    visible: true
 
 documentation_engine:
-  visible: true
+    visible: true
 
 documentation_url:
-  visible: true
+    visible: true
 
 use_tools:
-  choices:
-    - bandit
-    - conda
-    - coverage
-    - ruff
-    - makim
-    - mccabe
-    - mypy
-    - pre-commit
-    - pytest
-    - hypothesis
-    - shellcheck
-    - vulture
-  visible: true
+    choices:
+        - bandit
+        - conda
+        - coverage
+        - ruff
+        - makim
+        - mccabe
+        - mypy
+        - pre-commit
+        - pytest
+        - hypothesis
+        - shellcheck
+        - vulture
+    visible: true
```

Comment:
@xmnlab Do you think we should have an alphabetical order here?

Reply:
I think so, alphabetical order looks good to me.